src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/setting/handler/PerformanceAnalyzerClusterSettingHandler.java

```diff
@@ -95,13 +107,15 @@ public int getCurrentClusterSettingValue() {
     * @return the cluster setting value
     */
     private Integer initializeClusterSettingValue(
```

Comment:
Have we tested this across 1/ multi-node scenarios, 2/ clusters with nodes on different versions of this code (one node with this change and another on a version before this change) ?

Reply:
Haven't tested this out yet, but the logic should work out. Additionally, the behavior for this feature would be consistent with the behavior for the others.